references/video_classification/README.md

```diff
@@ -97,6 +97,21 @@ Video resnet models:
 --val-crop-size 112 112
 ```
 
+### S3D
+
+The S3D model was trained similarly to the above but with the following changes on the default configuration:
```

Comment:
NIT: I know this is a default value, but I think we may want to put `--frame-rate 15` in the previous section (resnet) since I think this is a quite important parameters for video classification.

Reply:
Sounds good, let me add it above.